### PR TITLE
Feat/aggresive fixers

### PR DESCRIPTION
**Pull request by mthines**
New branch from master to clear all the formating. See https://github.com/NoA-Ignite-dk/eslint-config/pull/13 for history.

### CODE REVIEW

**Comment by mthines** [comment]
> We just removed `prettier` from `next-boilerplate`. Really fucked it for me, had to remove prettier from vs-code :D

Yeah, I don't quite get why we've done that. Maybe that's for another meeting :)

File: next.js [hunk 1]
```diff
@@ -6,4 +6,40 @@ module.exports = {
...
+					['^(state/)'],
+					['^(styles/)'],
+					['^(translations/)'],
+					['^(types/)'],
+					['^(utils/)'],
+					['^(assets/|(.*).(scss|css)$)'],
+					['^((./|../))'],
+				],
+			},
+		],
```
Comment (jdpnielsen) [inline next.js:33]: I think this is a better fit for an overwrite in specific projects - maybe good to add to the boilerplate.

File: index.js [hunk 2]
```diff
@@ -44,8 +44,12 @@ module.exports = {
 		'max-len': ['error', {
 			code: 140,
 			ignoreComments: true,
+			// SVG d properties
+			ignorePattern: 'd="([\\s\\S]*?)"',
```
Comment (jdpnielsen) [inline index.js:48]: Nice. Might also be nice to figure out how to do something for jsx 'text'

**Review by jdpnielsen — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (mthines) [reply index.js:48]: Totally!
For later :D

**Review by jdpnielsen — APPROVED** [review]

**Pull request merged**